### DISCUSSION TITLE
No font styles or font family variables set in the UI Kit

### DISCUSSION THREAD

**Discussion by robertomouracosta** [primefaces/community#3145] [Figma UI Kit]
My company just brought the Prime One UI Kit and after open it in Figma. I saw there is no font styles or font variables set. It says you guys decided to use Figma default font Inter. 

My question is: Since there is no font style or font family variables set how would you recommend to do when I want to use my custom font?

**Comment by robertomouracosta** [comment]
I think the question can be improved. 

How would you recommend to do if I want to use a custom font and apply it to all components?

**Comment by onursenture** [comment]
Hello, try plugins to change all fonts quickly and efficiently. I tested the scenario of switching from Inter to Lato across the entire document using Font Replacer Ultimate and encountered no issues.

[Font Replacer Ultimate Plugin](https://www.figma.com/community/plugin/1100808562040625391)

Beyond this, I recommend creating your own text styles if you want a more permanent font style solution. You can then use the Styles & Variables Organizer to scan the page and merge all Unlinked text uses with your custom text styles.

[Styles & Variables Organizer Plugin](https://www.figma.com/community/plugin/816627069580757929)
